docs/adr/0002-use-slf4j-for-logging.md

```diff
@@ -0,0 +1,31 @@
+# Use slf4j together with log4j2 for logging
+
+* Currently JabRef uses apache commons logging 1.2 for logging errors and messages. However, this is not compatible with java 9 and is superseeded by log4j.
+* SLF4J provides a facade for several logging frameworks, including log4j and supports already java 9
+* Log4j is already defined as dependency and slf4j has already been required by a third party dependency
+
+## Considered Alternatives
+
+* [Log4j2](https://logging.apache.org/log4j/2.x/)
+* [SLF4J](https://www.slf4j.org/)
+
+
+## Decision Outcome
+
```

Comment:
Please place the decision here. :-)

The next subsection is for a detailed elaboration.